implementations/rust/ockam/ockam_transport_uds/src/router/handle.rs

```diff
@@ -0,0 +1,176 @@
+use std::os::unix::net::SocketAddr;
+
+use ockam_core::{
+    async_trait, compat::sync::Arc, Address, AsyncTryClone, Mailbox, Mailboxes, Result,
+};
+use ockam_node::Context;
+use ockam_transport_core::TransportError;
+
+use crate::{
+    address_from_socket_addr, parse_socket_addr,
+    workers::{UdsListenProcessor, WorkerPair},
+    UDS,
+};
+
+use super::{UdsRouterRequest, UdsRouterResponse};
+
+/// A handle to connect to a [`UdsRouter`](crate::router::UdsRouter)
+///
+/// Dropping this handle is harmless
+pub(crate) struct UdsRouterHandle {
+    ctx: Context,
+    main_addr: Address,
+    api_addr: Address,
+}
+
+#[async_trait]
+impl AsyncTryClone for UdsRouterHandle {
+    async fn async_try_clone(&self) -> Result<Self> {
+        // TODO @ac 0#UdsRouterHandle.async_try_clone.detached
+        // in:  n/a - DenyAll?
+        // out: n/a - DenyAll?
+        let mailboxes = Mailboxes::new(
```

Comment:
Should be DenyAll for incoming and outgoing

Reply:
Could you please refer to `tcp` crate and see what are the access controls in similar cases?